src/script/backup/BackupRepository.ts

```diff
@@ -78,8 +77,6 @@ export class BackupRepository {
   constructor(
     backupService: BackupService,
     conversationRepository: ConversationRepository,
-    private readonly clientState = container.resolve(ClientState),
-    private readonly userState = container.resolve(UserState),
```

Comment:
The repository doesn't depend on state now, instead the client and user are injected in the methods that require them